<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDatasetWidget</class>
 <widget class="QWidget" name="AddDatasetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>导入数据集</string>
  </property>
  <widget class="QPushButton" name="DatasetOK">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>420</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>601</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="title">
       <string>创建信息</string>
      </property>
      <widget class="QLabel" name="LabelDatasetID">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>数据集ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="DatasetID">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>D1</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="LabelDatasetDescription">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>90</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>数据集描述</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="DatasetDescription">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>561</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="LabelDatasetName">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>40</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>数据集名称</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="DatasetName">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="title">
       <string>数据集导入</string>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>90</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>数据集路径</string>
       </property>
      </widget>
      <widget class="QToolButton" name="findPath">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>150</y>
         <width>60</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>561</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>需要选定数据集所在文件夹路径（路径中仅含一个数据集)，不支持.zip、tar.gz等压缩包形式的数据导入，图片格式支持png. jpg. jpeg. bmp格式，文件夹名为需要分类的类名，输入限定为英文字符，不可包含空格、中文或特殊字符。</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="datasetPath">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>150</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>onClickOK()</slot>
 </slots>
</ui>
